Cargo.lock

```diff
@@ -551,21 +535,6 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
-[[package]]
-name = "chrono"
```

Comment:
I'm genuinely shocked `oauth2` was the only crate in all our dependencies using `chrono`!